contrib/skopeoimage/stable/Containerfile

```diff
@@ -0,0 +1,44 @@
+# stable/Containerfile
+#
+# Build a Skopeo container image from the latest
+# stable version of Skopeo on the Fedoras Updates System.
+# https://bodhi.fedoraproject.org/updates/?search=skopeo
+# This image can be used to create a secured container
+# that runs safely with privileges within the container.
+#
+FROM registry.fedoraproject.org/fedora:latest
+
+# Don't include container-selinux and remove
+# directories used by dnf that are just taking
+# up space.
+RUN dnf -y update && \
+    rpm --setcaps shadow-utils 2>/dev/null && \
```

Comment:
Do you happen to have a reference to why this is necessary? I could only find RHEL bugs and at least https://bugzilla.redhat.com/show_bug.cgi?id=1995337#c3 (private) suggests this should have been fixed, for RHEL.

(Absolutely non-blocking, we didn’t have a reference before.)

Reply:
This is something Dan knows about, we do the exact same thing in the podman and buildah images.  It replaces an old `restorecon` command we had for a long while.  I think I might be able to dig up a reference discussion at least (which may be important for your bugzilla finding)...

Reply:
...ahh it was Nalin, https://github.com/containers/podman/pull/14437#discussion_r887295318  and is Fedora specific.  Though on second reading, we probably don't need this in a skopeo image.  I'll remove it to save anybody from wondering or looking for related issues/documentation.

Reply:
… and looking at https://src.fedoraproject.org/container/fedora-container-image/blob/rawhide/f/image-build.conf (although the F30 references are _quite_ suspect, so I might be looking at the wrong place), the necessary tar options seem to be missing. Thanks, that explains things.

---

As a maintainer, my primary concern with a workaround is being able to maintain it over time. I.e. “how do I know that the workaround broke, and how can I verify that it was fixed again”. This thread definitely helps.

Reply:
@jnovy do you know if https://bugzilla.redhat.com/show_bug.cgi?id=1995337#c3 still affects Fedora, or where to confirm?